src/MoBi.Presentation/Presenter/ParameterValuesPresenter.cs

```diff
@@ -119,6 +119,8 @@ private IMoBiCommand addParameterValuesForObjectPaths(IReadOnlyList<ObjectPath>
          if (allSkipped.Any())
             _dialogCreator.MessageBoxInfo(AppConstants.Captions.BuildingBlockAlreadyContains(allSkipped.Select(x => x.PathAsString).ToList()));
 
+         _dialogCreator.MessageBoxInfo(AppConstants.Warnings.CheckDimensionOfAddedParameters);
```

Comment:
Could it happen that there are no parameter values added? If so, I'd skip the dialog.

Try to select nothing in the selection dialog, or select one, add it, then add the same one again. It will be skipped.

If nothing gets added to the BB let's skip the notifications